web/frontend/src/pages/ballot/components/Select.tsx

```diff
@@ -60,7 +60,14 @@ const Select: FC<SelectProps> = ({ select, answers, setAnswers, language }) => {
     setTitles(select.Title);
   }, [select]);
 
-  const choiceDisplay = (isChecked: boolean, choice: string, choiceIndex: number) => {
+  const choiceDisplay = (isChecked: boolean, choice: string, url: string, choiceIndex: number) => {
+    const prettyChoice = url ? (
+      <a href={url} style={{ color: 'blue', textDecoration: 'underline' }}>
```

Comment:
Shouldn't the `style` be in double quotes? This creates an object and then does `toString` on it, doesn't it? But I haven't tested it.

```suggestion
      <a href={url} style="color: 'blue', textDecoration: 'underline'">
```

Reply:
I followed another example from the existing code as I'm not very familiar with that :woman_shrugging: when I'm working on the CSS I'll probably come back to it anyway

Reply:
I'm pretty sure it's correct what I write - but I was already surprised with other stuff in react. So probably Ahmed will know better.